### PR TITLE
Add binarytree

### PR DESCRIPTION
**Pull request by cshaley**
Create conda forge package for binarytree, educational library for learning about binary trees.

### CODE REVIEW

**Comment by cshaley** [comment]
@joowani Are you okay with being a maintainer of this recipe?

File: recipes/binarytree/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - binarytree
```
Comment (CJ-Wright) [inline recipes/binarytree/meta.yaml]: You might also try importing the exceptions

Comment (cshaley) [reply recipes/binarytree/meta.yaml]: Just imported the exceptions module as a whole.  Did you want to import them one by one instead?

Comment (CJ-Wright) [reply recipes/binarytree/meta.yaml]: No that's fine, it's just a nice way to doubly check that the namespace installed correctly.

**Comment by joowani** [comment]
@cshaley Sure I don't mind!

**Pull request merged**